crates/database/src/auction_prices.rs

```diff
@@ -45,6 +45,16 @@ pub async fn fetch(
     Ok(prices)
 }
 
+pub async fn fetch_latest_prices(ex: &mut PgConnection) -> Result<Vec<AuctionPrice>, sqlx::Error> {
+    const QUERY: &str = r#"
+        SELECT * FROM auction_prices WHERE auction_id = (
+            SELECT MAX(auction_id)
+            FROM auction_prices
+        )
```

Comment:
nit: extra tabs should be avoided like it is done for other similar queries.
```suggestion
SELECT * FROM auction_prices WHERE auction_id = (
    SELECT MAX(auction_id)
    FROM auction_prices
)
```